atom.mk

```diff
@@ -92,7 +92,9 @@ LOCAL_GENERATED_SRC_FILES := version.c
 LOCAL_COPY_FILES := scripts/pv_e2fsgrow:lib/pv/pv_e2fsgrow \
```

Comment:
In general I think such change should refer to a spec or a docs page describing the new syntax ... can we make a doc entry for the new disks approach in /docs project so it appears on docs.pantahub.com  ... which we can then refer to in this commit? we can keep that page very minimal at first, but at best it should be a bit useful from the start ....  so we can unblock the landing of course

Reply:
In case you don't know where to put it, I'd suggest a new page in "Pantavisor reference" for disks and also add the disk key to "Container Configuration" page

Reply:
Thanks for the hint. Adding the docs and checking with local docker.

Reply:
https://gitlab.com/pantacor/docs/-/merge_requests/82